### PR TITLE
Use correct L10N files for jsconfig

### PR DESCRIPTION
**Pull request by rullzer**
To test:

1. set lanugage to other than english
2. open date picket (for example when setting expiration date on a share)

Before: still english
Now: Properly translated

### CODE REVIEW

File: core/Controller/OCJSController.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ class OCJSController extends Controller {
 	 */
 	public function __construct($appName,
 								IRequest $request,
-								IL10N $l,
+								IFactory $l,
```
Comment (nickvergessen) [inline core/Controller/OCJSController.php]: mind renaming it to something other than `$l` its quite confusing, because `$l` is reserved for `IL10N`

**Review by nickvergessen — APPROVED** [review]
"Moments when I have DI magic" +1

File: core/Controller/OCJSController.php (hunk 1 above)
Comment (rullzer) [reply core/Controller/OCJSController.php]: Done

File: core/Controller/OCJSController.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ class OCJSController extends Controller {
 	 */
 	public function __construct($appName,
 								IRequest $request,
-								IL10N $l,
+								IFactory $l10Factory,
```
Comment (nickvergessen) [inline core/Controller/OCJSController.php]: *n

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**